src/test/kotlin/ecommerce/controller/AuthControllerTest.kt

```diff
@@ -0,0 +1,292 @@
+package ecommerce.controller
+
+import ecommerce.auth.JwtTokenProvider
+import ecommerce.controller.api.AuthController
+import ecommerce.dao.JdbcMemberDAO
+import ecommerce.dto.AuthResponse
+import ecommerce.dto.LoginForm
+import ecommerce.dto.MemberResponse
+import ecommerce.dto.RegisterForm
+import ecommerce.model.Member
+import ecommerce.service.AuthService
+import io.restassured.RestAssured
+import io.restassured.http.ContentType
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.jdbc.core.JdbcTemplate
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class AuthControllerTest {
+    @Autowired private lateinit var jdbcTemplate: JdbcTemplate
+
+    @Autowired private lateinit var jdbcMemberDAO: JdbcMemberDAO
+
+    @Autowired private lateinit var jwtTokenProvider: JwtTokenProvider
+    private lateinit var authService: AuthService
+    private lateinit var controller: AuthController
+
+    @BeforeEach
+    fun setUp() {
+        jdbcMemberDAO = JdbcMemberDAO(jdbcTemplate)
+        authService = AuthService(jdbcMemberDAO, jwtTokenProvider)
+
+        jdbcTemplate.execute("DROP TABLE member CASCADE")
+        jdbcTemplate.execute(
+            """CREATE TABLE member
+            (
+                id       LONG         NOT NULL AUTO_INCREMENT,
+                email    VARCHAR(255) NOT NULL UNIQUE,
+                password VARCHAR(255) NOT NULL,
+                role     VARCHAR(255) DEFAULT NULL,
+                PRIMARY KEY (id)
+            );""",
+        )
+
+        val query =
+            """INSERT INTO member (email, password, role) VALUES ( 'san@htc.com', 'san1234', 'admin');
+            INSERT INTO member (email, password, role) VALUES ( 'dan@htc.com', 'dan1234', 'admin');
+            INSERT INTO member (email, password) VALUES ( 'ann@htc.com', 'ann1234');
+            INSERT INTO member (email, password) VALUES ( 'min@htc.com', 'min1234');"""
+        jdbcTemplate.batchUpdate(query)
```

Comment:
It would be great to find out other ways of adding test fixtures before each tests, and compare them.
After that, let's see if there are any better options we could choose for our situation.
(It's also fine to keep as is, but I want to know what you think!)